lib/tasks/migrate/20250326140624_add_channel_to_articles.rake

```diff
@@ -0,0 +1,30 @@
+# bundle exec rake check:migrate:add_channel_to_articles
```

Comment:
@melsawy if/when you have some time, can you please take a look at this file and let me know what you think?

Reply:
LGTM. I tested it against a live dump with `345609` FactCheck and it finished in `0m`. I also checked the live data, and it has `639838` articles, so I believe we are on the safe side.
great work @vasconsaurus